openrouteservice/src/main/java/org/heigit/ors/api/responses/routing/gpx/GPXMetadataExtensions.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of Openrouteservice.
+ *
+ * Openrouteservice is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU Lesser General Public License as published by the Free Software Foundation; either version 2.1
+ * of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with this library;
+ * if not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.heigit.ors.api.responses.routing.gpx;
+
+import org.heigit.ors.api.requests.routing.RouteRequest;
+import org.heigit.ors.api.util.SystemMessage;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(name = "extensions")
+public class GPXMetadataExtensions {
+    @XmlElement(name = "system-message")
```

Comment:
old habits, XML naming convention used to be lowercase with hyphens or camel case. But our GPX format is a mess anyway, there is a "distance-units" and a "build_date" element in the extension, and several elements in the global extensions belong in the metadata anyway if you ask me. In JSON objects I'd normally use camel case, because it's meant to be handled in javascript, but there are already underscores in use (more consistently, afaik), so I went for that. 
To clean this up we'd need API v3. Maybe we should collect things like that and do a planned cleanup update at some point?  
As for now, I'm really not used to underscores in XML anyway, probably because I've been in the web and javascript world too much. Hyphens feel like better style to me, though we should maybe discuss this and add to our overall coding style guide (which we don't have yet, yay).